fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneAutoCompletedMatchesTest.java

```diff
@@ -63,6 +63,21 @@ void searchForQua(String text, int numAdditionalTokens, List<String> expected) t
         assertComputeAllMatches("qua", Collections.emptyList(), ImmutableSet.of("qua"), text, numAdditionalTokens, expected);
     }
 
+    static Stream<Arguments> searchForCjk() {
+        return ImmutableList.of(
+                Arguments.of("世上无难事", "\"上无\"", Arrays.asList("上", "无"), null, 0, ImmutableList.of("上无")),
+                Arguments.of("世上无难事", "\"上无\"", Arrays.asList("上", "无"), null, 2, ImmutableList.of("上无难事")),
+                Arguments.of("世の中に難しいことなんてないよ", "\"の中に\"", Arrays.asList("の", "中", "に"), null, 3, ImmutableList.of("の中に難しい")),
+                Arguments.of("世ノ中ニシイコトナンテナ", "\"シイコ\"", Arrays.asList("シ", "イ", "コ"), null, 3, ImmutableList.of("シイコトナン")),
+                Arguments.of("세상에 어려운 일은 없다", "\"어려\"", Arrays.asList("어", "려"), null, 2, ImmutableList.of("어려운 일"))).stream();
```

Comment:
All of these have a single element in the `expected`, are we missing important coverage by not having something with multiple auto-completes.

Reply:
this is testing whether the matcher matches the right part of the completion result, given an input, an analyzer, and the requested number of additional tokens, so I think single element tests are ok.

The actual tests for Lucene auto completion search is in `LuceneIndexTest.java`. For those, I have cases with multiple elements. I can add more to increase coverage.

Reply:
Or if you mean multiple occurrences within the same input, yes I could add that case as well